### DISCUSSION TITLE
Can this be installed in a local implementation of Automatic1111?

### DISCUSSION THREAD

**Discussion by SirTrippsalot** [TheLastBen/fast-stable-diffusion#239] [Q&A] [answered]
I only see instructions on using this with COLAB.
Can this be installed locally? If so, are there instructions somewhere on integrating this with a local install of automatic1111?

**Comment by TheLastBen** [comment, marked as answer]
Soon there will be a local version of the colab

**Reply by SirTrippsalot** [reply]
Thanks for the info, and for your work :)

**Reply by artificialguybr** [reply]
Kind of a silly question, but would the local version work on GPU Rent or even Kaggle? @TheLastBen
